src/main/java/org/jabref/gui/openoffice/StyleSelectDialogView.java

```diff
@@ -74,11 +77,16 @@ public StyleSelectDialogView(StyleLoader loader) {
         setResultConverter(button -> {
             if (button == ButtonType.OK) {
                 viewModel.storePrefs();
+                System.out.println("Selected CSL Style Index: " + viewModel.getSelectedStyleIndex());
```

Comment:
This is also covered by our tests. @subhramit Please try to have a look at the failing tests and try to understand what's going on here and how you could fix it.

![image](https://github.com/subhramit/jabref/assets/1366654/19b57134-eba9-4209-8dca-91ea5bbb34e3)

You are very welcome to create a `faq.md` at https://devdocs.jabref.org/code-howtos/ showing how to resolve "common" errors. The case of using `System.out.println` is such a common case. The answer of the FAQ should link to https://devdocs.jabref.org/code-howtos/logging.html.

Reply:
Sure, will do!
I also suspect there could be failing tests because my branch is outdated and (somehow) has many conflicts to resolve.

Reply:
[Note for future reference] The failing test was because of:
1. A class purely containing business logic being put at some level inside the ```models``` directory. Fix: moved it to a subdirectory under ```logic```.
2. Use of ```System.out.println``` instead of the logger for logging.